extensions/sentry/package.json

```diff
@@ -20,11 +21,19 @@
   ],
   "preferences": [
     {
+      "description": "Auth Token to communicate with Sentry",
       "name": "token",
+      "required": true,
       "title": "Auth Token",
-      "type": "password",
-      "description": "Auth Token to communicate with Sentry",
-      "required": true
+      "type": "password"
+    },
+    {
+      "description": "URL of the Sentry instance to connect with",
+      "name": "url",
+      "required": true,
+      "default": "https://sentry.io",
```

Comment:
Could we add a placeholder with the default URL here?

Reply:
Can we not make it required and instead use the non-hosted version as default? I think that applies to most people.

Reply:
Thanks for the feedback

both of your comments are resolved